vendor/github.com/vmware/go-vcloud-director/v2/govcd/upload.go

```diff
@@ -105,19 +105,22 @@ func uploadFile(client *Client, filePath string, uDetails uploadDetails) (int64,
 }
 
 // Create Request with right headers and range settings. Support multi part file upload.
```

Comment:
If I go it right, we should not commit changes to vendor which are linked to personal repo

Reply:
It's just for testing (and easier build) as we had these recent discussions.

Reply:
Yep, I don't know what is for testing and what is not. So don't forget to delete before merging.

Reply:
I will tag govcd at first and then repoint. You can ignore checking `vendor` directory safely.